src/lib/urlUtils.js

```diff
@@ -1,15 +1,16 @@
 /* eslint-disable import/prefer-default-export */
+/* eslint-disable no-useless-escape */
 // regex to match a valid url, it must start with http:// or https://, have at least one dot, and not end with a dot or a space
-const VALID_URL_REGEX = /^https?:\/\/.*\.[^ |^.]/;
+export const VALID_URL_REGEX = /(?<url>(?<scheme>http(?:s)?|ftp(?:s)?|sftp):\/\/(?:(?<user>[a-zA-Z0-9._]+)(?:[:](?<password>[a-zA-Z0-9%._\+~#=]+))?[@])?(?:(?:www\.)?(?<host>[-a-zA-Z0-9%._\+~#=]{1,}\.[a-z]{2,6})|(?<ip>(?:[0-9]{1,3}\.){3}[0-9]{1,3}))(?:[:](?<port>[0-9]+))?(?:[\/](?<path>[-a-zA-Z0-9'@:%_\+.,~#&\/=()]*[-a-zA-Z0-9@:%_\+.~#&\/=()])?)?(?:[?](?<query>[-a-zA-Z0-9@:%_\+.~#&\/=()]*))*)/ig;
```

Comment:
There is no way we should be allowing anything other than http/s

Reply:
ok i will remove the ftp ones